.github/workflows/gh-pages.yaml

```diff
@@ -48,6 +48,13 @@ jobs:
       - name: Install dependencies
         run: python3 -m pip install -r docs/requirements.txt
 
+      - name: Find and replace "%pip install cleanlab" in .ipynb files
+        uses: jacobtomlinson/gha-find-replace@v2
+        with:
+          find: "%pip install cleanlab"
+          replace: "%pip install git+https://github.com/${{ github.repository }}.git@${{ github.sha }}"
```

Comment:
Colab should only be installing from github when this is not a tagged version release.  If this is the latest stable version, then do not need to replace. If this is an older tagged version that is no longer the latest stable version, then we want to replace with:   
`"%pip install cleanlab==X.Y.Z" with X.Y.Z indicating the version number."

Reply:
done. added additional steps after the one above to handle this.